scripts/fetch.mjs

```diff
@@ -1,120 +1,167 @@
+      // Check if the constant has any filters
+      if ("filters" in constant && constant.filters.length > 0) {
+        // Apply filter formatting
+        constant.filters.forEach(filter => {
+          chainValue = applyFilter(chainValue, filter, wiki);
+        });
+      }
+
+      // Update 
+      constantsDict[`{{ ${constant.tpl} }}`] = chainValue;
     });
   })
   .catch(function (e) {
     console.log(e);
-    console.error(
-      "Error connecting! Check your node URL and make sure its websockets are open, secure if remote (wss), and allow RPC from all."
-    );
     process.exit(1);
   });
 
 let v = setInterval(function () {
-  if (Object.keys(filledDict).length === Object.keys(replacements).length) {
+  // Verify the correct amount of expected constant values are generated
+  if (Object.keys(constantsDict).length === Object.keys(constants).length) {
     clearInterval(v);
-    const content = JSON.stringify(filledDict, null, 2);
 
-    fs.writeFileSync("./scripts/computed-dict.json", content, { encoding: "utf8" });
-    console.log("Save dict values successfully!");
+    // Write updated computed-dict.json file
+    const content = JSON.stringify(constantsDict, null, 2);
+    writeFileSync("./scripts/computed-dict.json", content, { encoding: "utf8" });
+    console.log("Updated global constants in computed-dict.json");
+
+    // If the injection flag is present, inject the constants into the doc html
+    if (argv.inject) {
+      // Template options for replace-in-file
+      const options = {
+        files: [`${argv.rootDir}/docs/**/**/*.html`],
+        from: Object.keys(constantsDict).map((el) => {
+          return new RegExp(el, "ig");
+        }),
+        to: Object.values(constantsDict),
+      };
+
+      console.log("Replacement configuration: ");
+      console.log(options);
+
+      try {
+        let results = replace.sync(options);
+
+        const changedFiles = results
+          .filter((result) => result.hasChanged)
+          .map((result) => result.file);
+        console.log("Modified files:", changedFiles);
+
+        let from = [/\{\{ kusama: [\s\S]+? :kusama \}\}/gim, /\{\{ polkadot: [\s\S]+? :polkadot \}\}/gim];
+        let to =
+          argv.rootDir.indexOf("kusama-guide") !== -1
+            ? [(match) => match.replace("{{ kusama: ", "").replace(" :kusama }}", ""), ""]
+            : ["", (match) => match.replace("{{ polkadot: ", "").replace(" :polkadot }}", "")];
+        let results2 = replace.sync({
+          files: [`${argv.rootDir}/docs/**/**/*.html`],
+          from: from,
+          to: to,
+        });
+        const changedFiles2 = results2
+          .filter((result) => result.hasChanged)
+          .map((result) => result.file);
+        console.log("Modified files for kusama/polkadot difference:", changedFiles2);
+      } catch (error) {
+        console.error("Error occurred:", error);
+        process.exit(1);
+      }
```

Comment:
This is the only logic that was unique to `inject.mjs` that wasn't already in `fetch.mjs`, everything else was duplicated.  I haven't verified this logic but it produces the same results as the previous state.